routes/artist.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express"); 
+const router = express.Router(); 
+const artist = require("../models/newArtist");
+const Artist = require("../models/newArtist");
+
+router.post("/", (req, res) => {
+    let userid = req.body.userid; 
+    let name = req.body.name; 
+    let nationality = req.body.nationality; 
+    let medium = req.body.medium;
+    let artist = new Artist({userid: userid, name: name, nationality: nationality, medium: medium});
+    artist.save();
+    res.status(201).send("New Artist Added to Your List!");
+});
```

Comment:
Remember, some error handling here would be great, maybe the database connection got interupted or timed out, so we can pass a 500 with an error back in the response so users know what happened